lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -229,4 +233,17 @@ export default class GutenbergEditorComponent extends AsyncBaseContainer {
 		);
 		return hasPublishButton && ! hasRevertButton;
 	}
+
+	async viewPublishedPostOrPage( { reloadPageTwice = false } = {} ) {
+		const viewPostSelector = By.css( '.components-notice__content a' );
+		const driver = this.driver;
+
+		await driverHelper.waitTillPresentAndDisplayed( this.driver, viewPostSelector );
+
+		let url = await driver.findElement( viewPostSelector ).getAttribute( 'href' );
```

Comment:
we only did this previously since it opened it a new window and it saved us messing around
since this doesn't open a new window can we just click the link? Simpler and also more representative of real use